doc/reference/kernel/other/float.rst

```diff
@@ -193,6 +193,31 @@ when there is no need to do so.
 When the thread again needs to use the floating point registers it can re-tag
 itself as an FPU user or SSE user by calling :cpp:func:`k_float_enable()`.
 
+
+RISCV architecture
+------------------
+
+On the RISCV architecture, the kernel treats each thread as a non-user
```

Comment:
For clarity, how about:
```
On the RISCV architecture, the kernel treats each thread as
a non-FPU user unless the thread is tagged as an FPU user
using one of the following techniques: 
```

What are the consequences if a thread does floating point operations
but was not tagged as an FPU user?  Context switches won't be
saving/restoring the FP registers so bad things will happen, right?
If so, this should be emphasized here.

Reply:
Thanks for reviewing, I will update the document later according to your suggestion.

Reply:
Oh, so the default for RISCV is all threads are treated as FPU users?  The docs should certainly say that.